### PR TITLE
update homepage view/drop defaultCurrency

### PR DESCRIPTION
**Pull request by AlicjaSzu**
I want to merge this change because...

- it displays on homepage view information per channel in: Sales, Orders, Top Products

- it removes unneeded defaultCurrency and currencySymbol props
- it removes money spent filter on customer view

**PR intended to be tested with API branch:** feature/multichannel-multicurrency

### Screenshots

Screenshot 2020-10-29 at 00 16 12

### Pull Request Checklist

1. [x] All visible strings are translated with proper context.
1. [x] All data-formatting is locale-aware (dates, numbers, and so on).
1. [x] Translated strings are extracted.
1. [x] Number of API calls is optimized.
1. [x] The changes are tested.
1. [x] Data-test are added for new elements.
1. [x] Type definitions are up to date.
1. [ ] Changes are mentioned in the changelog.

### Test environment config

API_URI=https://feature-multichannel-multicurrency.api.saleor.rocks/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-1456 Drop defaultCurrency in frontend](https://app.clickup.com/t/2549495/SALEOR-1456)

**Comment by patrys** [comment]
Task linked: [SALEOR-1411 Implement new homepage view for multichannel homeview in dashborad](https://app.clickup.com/t/2549495/SALEOR-1411)

**Review by mmarkusik — APPROVED** [review]

File: src/home/views/index.tsx [hunk 1]
```diff
@@ -25,6 +37,9 @@ const HomeSection = () => {
           topProducts={maybe(() =>
             data.productTopToday.edges.map(edge => edge.node)
           )}
+          channelChoices={channelChoices}
+          channelValue={channelChoice}
+          onChannelChange={useCallback(value => setChannelChoice(value), [])}
```
Comment (dominik-zeglen) [inline src/home/views/index.tsx:42]: I believe this breaks rule of hooks
https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

File: src/products/views/ProductList/filters.ts [hunk 2]
```diff
@@ -207,15 +208,18 @@ export function getFilterVariables(
...
     collections: params.collections !== undefined ? params.collections : null,
     isPublished:
       params.status !== undefined
         ? params.status === ProductStatus.PUBLISHED
         : null,
-    price: getGteLteVariables({
-      gte: parseFloat(params.priceFrom),
-      lte: parseFloat(params.priceTo)
-    }),
+    price: channel
```
Comment (dominik-zeglen) [inline src/products/views/ProductList/filters.ts:217]: Is there a situation where channel is not chosen?
Don't have staging to check it 🤷‍♂️

**Review by dominik-zeglen — APPROVED** [review]

**Review by dominik-zeglen — CHANGES_REQUESTED** [review]
Oops, wrong button

File: src/products/views/ProductList/filters.ts (hunk 2 above)
Comment (AlicjaSzu) [reply src/products/views/ProductList/filters.ts:217]: it's for situation when channel is "loading"

**Review by dominik-zeglen — APPROVED** [review]

**Pull request merged**